src/pylorax/api/dnfbase.py

```diff
@@ -56,7 +56,10 @@ def get_base_object(conf):
     dbc.reposdir = [repodir]
     dbc.install_weak_deps = False
     dbc.prepend_installroot('persistdir')
-    dbc.tsflags.append('nodocs')
+    # this is a weird 'AppendOption' thing that, when you set it,
+    # actually appends. Doing this adds 'nodocs' to the existing list
+    # of values, over in libdnf, it does not replace the existing values.
+    dbc.tsflags = ['nodocs']
```

Comment:
For the record: internally changing the type of something coming from a 3rd party looks like a bad idea. I know it works b/c both are iterators but imagine going to dnf sources/docs where this is a tuple and then seeing it used as a list.

Not sure if it can lead to more severe issues in the future but definitely looks odd.

Reply:
It's not actually anything like that simple. It's not as simple as saying any of these things "are" lists or tuples, and it doesn't exactly work "because both are iterators", it works [because dnf is explicitly set up to accept them being set as either lists or tuples](https://github.com/rpm-software-management/dnf/blob/master/dnf/conf/config.py#L242). Note that's in the middle of a rather complicated `__getattr__` which, yes, in DNF 3.6 ultimately returns a tuple (but in all previous DNFs, you got a list).

We could set it as a tuple, sure, but it's not at all obviously "better". I went with list because it's more compatible with ancient yum/dnf which would not accept setting these as tuples, though that's probably not a major concern.